src/main/java/triss/Triss.java

```diff
@@ -0,0 +1,429 @@
+package triss;
+
+import triss.exception.TrissException;
+import triss.task.Deadline;
+import triss.task.Event;
+import triss.task.Task;
+import triss.task.Todo;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Triss {
+
+    /** Logo shown during startup */
+    private static final String LOGO = "████████ ██████  ██ ███████ ███████ \n" +
+            "   ██    ██   ██ ██ ██      ██      \n" +
+            "   ██    ██████  ██ ███████ ███████ \n" +
+            "   ██    ██   ██ ██      ██      ██ \n" +
+            "   ██    ██   ██ ██ ███████ ███████ \n";
+
+    /** String of underscores to separate user input and Triss output */
+    public static final String SEPARATOR_LINE = "____________________________________________________________";
+
+    /** Boolean to track if user has said "bye" */
+    private static boolean hasUserSaidBye = false;
```

Comment:
If these variables are specific to this Triss class/object, why not consider making all methods/some variables non static?

Reply:
Thanks, will try to decrease number of static variables.